### DISCUSSION TITLE
Caffeine when playing youtube

### DISCUSSION THREAD

**Discussion by sitolam** [prasanthrangan/hyprdots#1640] [Q&A] [answered]
How can I stop my screen locking and sleeping when playing a youtube video or other media?

**Comment by kRHYME7** [comment]
use hypridle or swayidle.

**Reply by sitolam** [reply]
I'm using that like this: `exec-once = swayidle -w timeout 300 'swaylock; hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' timeout 600 'systemctl suspend' # lock & turn off monitor after 5 mins, suspend after 10 mins // install swayidle`. But it still locks my screen when watching youtube.

**Reply by kRHYME7** [reply, marked as answer]
the waybar got an Idle inhibitor. You can Click the clock glyph.


Or you can install a package named sway audio idle inhibitor

**Reply by sitolam** [reply]
> the waybar got an Idle inhibitor. You can Click the clock glyph.
> 
> Or you can install a package named sway audio idle inhibitor

I used the `sway audio idle inhibitor` it works great. Thank you
